src/client/pages/Checkout/CheckoutPayment/ContactInformation/ContactInformation.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Headline } from 'components/Headline/Headline'
+import { InputField } from 'components/inputs'
+import { useTextKeys } from 'utils/textKeys'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useFeature, Features } from 'utils/hooks/useFeature'
+import { MEDIUM_SMALL_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { Divider } from '../../shared/Divider'
+
+type ContactInfoData = {
+  data: {
+    firstName?: string
+    lastName?: string
+    ssn?: string
+    email?: string
+  }
+}
```

Comment:
Why not just make the `props` object flat 🤔 ?

```suggestion
type ContactInfoData = {
  firstName?: string
  lastName?: string
  ssn?: string
  email?: string
}
```

Reply:
Haha good point! I forgot I had left it that way, the type looked completely different from the start 😅

Reply:
Haha true that! I forgot to remove it after I simplified the typing for this component. Thank you for catching it :D